src/components/Main.js

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from 'react'
+import {FiBookmark} from  "react-icons/fi";
+import {CgCloseO} from "react-icons/cg";
+
+
+export default function Main() {
```

Comment:
лучше обходить мимо дефолтных экспортов - тогда вс-код будет тебе подсказывать импорты лучше, ну и есть еще другие плюсы - можно почитать про сравнение экспортов
```suggestion
export function Main() {
```